sqlalchemy_mixins/serialize.py

```diff
@@ -8,7 +8,7 @@ class SerializeMixin(InspectionMixin):
 
     __abstract__ = True
 
-    def to_dict(self, nested=False, hybrid_attributes=False):
+    def to_dict(self,nested =False, hybrid_attributes=False, exclude = []):
```

Comment:
Looks good. A minor change. It is generally a bad practice to have a mutable value as a default value. Set it to `None` then initialize later like so:
```
if exclude is None:
    exclude = []
```

Reply:
Also in `serialize.pyi`, don't forget to update the signature of the method to:
```
def to_dict(self, nested: bool = False, hybrid_attributes: bool = False, exclude: Optional[list] = None) -> dict: ...
```

Reply:
Okay

Reply:
I put the default for exclude as None and did something like this 
```         
         if exclude is None:
              view_cols = self.columns
         else :
              view_cols = filter(lambda e: e not in exclude, self.columns)
```
I could just have done 
```
if exclude is None:
    exclude = []
```
and used filter on that but I thought since it is an optional argument that has niche use case it will add to the complexity